src/sentry/replays/endpoints/project_replay_details.py

```diff
@@ -44,3 +46,18 @@ def get(self, request: Request, project: Project, replay_id: str) -> Response:
             return Response(status=404)
         else:
             return Response({"data": response[0]}, status=200)
+
+    def delete(self, request: Request, project: Project, replay_id: str) -> Response:
```

Comment:
wondering if anyone should be able to delete a replay, or if just admins or above. don't think it matters too much for now, but something we might want to add later.

Reply:
from slack discussions, sounds like being open is good for now.

Reply:
could also add an audit log, but i don't think its necessary atm